main/src/library/InputStream.flix

```diff
@@ -0,0 +1,89 @@
+namespace InputStream {
+    ///
+    /// Wraps the `java.io.InputStream.available()` method that returns the number of available bytes
+    ///
+    /// See the Java documentation for more details
+    ///
+    pub def available(stream: InputStream): Result[Int32, Error] & Impure = try {
+        import java.io.InputStream.available();
+
+        let InputStream(javaStream) = stream;
+
+        Ok(javaStream.available())
+    } catch {
+        case _: ##java.io.IOException => Err(IOException)
+    }
+
+    ///
+    /// Wraps the `java.io.InputStream.read()` method that reads a single byte from the stream.
+    ///
+    /// See the Java documentation for more details
+    ///
+    pub def readByte(stream: InputStream): Result[Int32, Error] & Impure = try {
```

Comment:
We usually go for short variable names, so `s` for stream, `b` for buffer, etc.

Reply:
Will change it!